axelor-sale/src/main/resources/domains/LoyaltyAccount.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<domain-models xmlns="http://axelor.com/xml/ns/domain-models"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://axelor.com/xml/ns/domain-models http://axelor.com/xml/ns/domain-models/domain-models_7.1.xsd">
+
+  <module name="sale" package="com.axelor.apps.sale.db"/>
+
+  <entity name="LoyaltyAccount">
+
+    <many-to-one name="partner" title="Partner" ref="com.axelor.apps.base.db.Partner"/>
+    <many-to-one name="company" title="Company" ref="com.axelor.apps.base.db.Company"/>
+    <decimal name="pointsBalance" title="Points balance" scale="0" precision="8" default="0"/>
```

Comment:
Is scale to 0 intended in domain ? I understand that we scale down to 0 in process/view, etc... But why in domain too ? (Equivalent to integer)